CITATION.cff

```diff
@@ -0,0 +1,47 @@
+cff-version: 1.2.0
+title: bokeh-a11y-audi
+message: >-
+  If you use this resource, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: "Frank"
+    family-names: "Elavsky"
+    orcid: "https://orcid.org/0000-0002-6849-5893"
```

Comment:
I added your ORCID @frankelavsky as I thought you'd prefer having this here (I do not so I skipped it for myself)

Reply:
Thanks!!